src/olympia/yara/models.py

```diff
@@ -1,41 +0,0 @@
-from django.db import models
```

Comment:
Does django need a models.py file to "see" the app as possibly having migrations?